docker/upload.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+import os
+import time
+import json
+
+images = ["ubuntu", "graphiteapp/graphite-statsd", "g4s8/artipie-base"]
+
+# Start artipie with preconfigured docker repo
+def start_artipie():
+    print("Starting artipie")
+    artipie_yml = """
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+  layout: flat
+"""
+    my_docker = """
+repo:
+  type: docker
+  storage:
+    type: fs
+    path: /var/artipie/data
+  permissions:
+    "*":
+      - "*"
+"""
+    os.makedirs("./configs", exist_ok=True)
+    f = open("./artipie.yaml", "w+")
+    f.write(artipie_yml)
+    f.close()
```

Comment:
@Sammers21 it could be wrapped in `with` construct to close resource properly (similar to try-with-resources in Java):
```python
with open('./artipie.yaml', 'w+') as f:
  f.write(artipie_yml)
```